test/sys/test_pthread.rs

```diff
@@ -3,5 +3,5 @@ use nix::sys::pthread::*;
 #[test]
 fn test_pthread_self() {
     let tid = pthread_self();
-    assert!(tid > 0);
+    assert!(tid != 0);
```

Comment:
Actually the man page for `pthread_self` says we should only use `pthread_equals` for comparing pthreads. If anything we shouldn't even check this value as [it's non-portable](https://stackoverflow.com/questions/16599463/what-is-the-value-range-of-thread-and-process-id#16599528).